### PR TITLE
Re-format rules

### PR DESCRIPTION
**Pull request by metalgearsloth**
Member when we weren't supposed to have a wiki?

- Puts it into the guidebook.
- Puts the wiki content ingame.

See https://github.com/marketplace/actions/mediawiki-edit-action for mediawiki stuff
The test page https://wiki.spacestation14.io/wiki/Server_Rules
The PR removing the old workflow https://github.com/space-wizards/space-station-14/pull/20385

We could probably make a new page for testing though not sure if I had perms. Realistically I would also just tell people refer to ingame going forward.

TODO for later:
- [ ] Make MRP stuff only show up on MRP
- [ ] Port the rest of the stuff over
- [ ] Add label saying it's MRP at the top of the rules
- [ ] Add exporter for remaining stuff to wiki

:cl:
- add: Rules are now in the guidebook.

### CODE REVIEW

File: Content.Client/Info/RulesAndInfoWindow.cs [hunk 1]
```diff
@@ -62,7 +63,12 @@ private void AddSection(Info info, string title, string path, bool markup = fals
 
         private static Control MakeSection(string title, string path, bool markup, IResourceManager res)
         {
-            return new InfoSection(title, res.ContentFileReadAllText($"/ServerInfo/{path}"), markup);
+            var text = res.ContentFileReadAllText($"/ServerInfo/{path}");
+            // I either call the entire guidebook-specific DocumentParsingManager... or I can just prune <Document>
```
Comment (metalgearsloth) [inline Content.Client/Info/RulesAndInfoWindow.cs:67]: Both options seem equally bad.

**Comment by Emisse** [comment]
thank you christian soldier

File: Resources/Prototypes/Guidebook/ss14.yml [hunk 2]
```diff
@@ -14,3 +14,8 @@
   id: Writing
   name: guide-entry-writing
   text: "/ServerInfo/Guidebook/Writing.xml"
+
+- type: guideEntry
```
Comment (TurboTrackerss14) [inline Resources/Prototypes/Guidebook/ss14.yml:18]: Considering the relevance of DA RULES. Might be a good idea to bump this all the way up to the top or just below basic controls.

Comment (metalgearsloth) [reply Resources/Prototypes/Guidebook/ss14.yml:18]: It is at the top, prototype ordering doesn't matter.

File: Resources/ServerInfo/Guidebook/Rules/AlertProcedure.xml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<Document>
+
+[color=#ff0000]Alert Procedure[/color]
+
+[color=#ff0000]MRP Only[/color]
+
+See https://wiki.spacestation14.io/wiki/Alert_Procedure
+
```
Comment (metalgearsloth) [inline Resources/ServerInfo/Guidebook/Rules/AlertProcedure.xml:8]: Guidebook doesn't support tables and Alert Procedure is probably unreadable without it, same with Space Law.

File: Resources/ServerInfo/Guidebook/Rules/SecurityCommandRules.xml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    - Brig times should generally not exceed 20 minutes unless stated otherwise in Space Law. Repeat offenders or antagonists may be permabrigged in accordance with Space Law.
+    - Security may choose to confiscate dangerous items (weapons, firearms) as well as items used to commission crimes or items that prove problematic in possession of the detainee (tools, insulated gloves, etc.).
+    - Detainees that die in your custody must be cloned unless they have been (legally) executed, suicide, or there is strong reason to believe they are an antagonist.
+    - Executions must be for an executable crime and approved by the Captain/Acting Captain, who will answer for approving it alongside Security's chain of command. Those who willingly attempt to damage/destroy or escape from the permabrig may be executed.
+    - Any prisoner may be borged with their consent. Borging may be offered as an alternative to execution, but cannot be forced if the prisoner is able to consent.
+    - Detainees in the brig have the right to know what they are being charged with, as well as basic medical aid, at least to the point they are no longer at risk of dying.
+</Document>
```
Comment (metalgearsloth) [inline Resources/ServerInfo/Guidebook/Rules/SecurityCommandRules.xml:34]: For whatever reason MRP and LRP had differences on this when realistically you'd think they'd be standardised / differences highlighted? E.g. max brig timer is different, when to execute is different, etc.

**Comment by metalgearsloth** [comment]
I looked at https://github.com/space-wizards/space-station-14/pull/6222 and implementing wiki links when the rules are already hosted on github seems like an unnecessary PITA considering we want everything ingame anyway.

**Comment by metalgearsloth** [comment]
wiki is a pain in the ass.

**Pull request closed without merging**